lib/mods/replace-window-functions.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+const nodeTypes = require('ts-morph').TypeGuards;
+const {updateImports} = require('../imports.js');
+const {
+  formatReplacement,
+  getOriginal,
+  isGlobal,
+} = require('../originals-helper.js');
+
+/**
+ * Replaces window function calls in the current node
+ * with the Get-Originals version
+ *
+ * @param {Node} node - a ts-morph AST Node
+ * @param {Object} imports - the current list of imports
+ */
+function replaceWindowFunctions(node, imports) {
+  if (nodeTypes.isCallExpression(node) && isGlobal(node.getExpression())) {
+    const symbol = node.getExpression().getSymbol();
+    const firstDeclaration = symbol.getDeclarations()[0];
+    const type = 'window';
+    const mod = 'Window';
+    let name = symbol.getName();
+
+    if (firstDeclaration) {
```

Comment:
When is this necessary?

Reply:
According to jest's `--coverage` flag, never